tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/basic/api/ScalarHolder.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.graphql.tck.apps.basic.api;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+/**
+ * Plain POJO with scalar fields
+ * @author Phillip Kruger (phillip.kruger@redhat.com)
+ */
+public class ScalarHolder {
+    private short shortPrimitive;
+        private Short shortObject;
+        
+        private int intPrimitive;
+        private Integer intObject;
+        
+        private long longPrimitive;
+        private Long longObject;
+
+        private float floatPrimitive;
+        private Float floatObject;
+        
+        private double doublePrimitive;
+        private Double doubleObject;
+    
+        private boolean booleanPrimitive;
+        private Boolean booleanObject;
+        
+        private char charPrimitive;
+        private Character charObject;
+
+        private char[] stringPrimitive;
```

Comment:
This might be a bug in my implementation, but I fail here with my schema returning `stringPrimitive: [Char]` instead of `stringPrimitive: String`.

I know that we have it documented that a char[] is supposed to map to a String (scalar).  Is that what the implementation should be doing?  Or should we change the doc and test to expect a `[Char]` schema type?

Reply:
Can I create an item on the agenda, then we discuss there ? Then we merge this, and if needed do another PR from the outcome of the discussion ?